nservicebus/messaging/routing.md

```diff
@@ -78,7 +78,7 @@ Unicast routing uses queues which are inherently point-to-point channels. Each e
 
 ### Multicast routing
 
-Multicast routing uses topics. Multiple receivers can subscribe to a single topic and each message published to the topic is copied by the messaging infrastructure and delivered to each interested receiver.
+Multicast routing uses topics or similar devices. Multiple receivers can subscribe to a single topic and each message published to the topic is copied by the messaging infrastructure and delivered to each interested receiver.
```

Comment:
> Multiple receivers can subscribe to a single topic and each message published to the topic is copied by the messaging infrastructure and delivered to each interested receiver.

Too long:

"Multiple receivers can subscribe to a single topic. In such case every published message is copied by the messaging infrastructure, and the copy is delivered to all interested receivers."